order/nulls.go

```diff
@@ -0,0 +1,9 @@
+package order
+
+// Nulls represents the position of nulls in an ordering of values.
+type Nulls bool
+
+const (
+	NullsLast  Nulls = false
+	NullsFirst Nulls = true
```

Comment:
Shouldn't this be NullsMin/NullsMax to match the method in order/which?

Reply:
I've found that this nulls stuff is cleaner and easier if as much of the code as possible reflects the "nulls first"/"nulls last" model in SPQ and SQL.

Reply:
I should say that you don't really see that here but you do in #5847.